src/vantage-ui-template/index.html

```diff
@@ -21,244 +21,10 @@
-              />
-              <path
-                id="Fill-7"
-                d="M216.11,37.026 C200.966,37.026 190.121,43.069 182.643,53.913 L182.643,36.899 L157.963,36.899 L157.963,130.197 L182.643,130.197 L182.643,100.095 C182.643,74.106 193.113,57.776 215.923,57.776 L217.044,57.776 L217.044,37.029 C216.741,37.027 216.431,37.026 216.11,37.026"
-              />
-              <path
-                id="Fill-8"
-                d="M48.982,107.707 C45.229,109.115 41.291,110.002 38.328,110.002 C30.102,110.002 24.68,105.142 24.68,93.923 L24.68,57.09 L49.982,57.09 L49.982,36.898 L24.68,36.898 L24.68,14.007 L0,14.007 L0,95.793 C0,121.408 14.584,133 36.646,133 C44.438,133 51.879,131.338 61.481,126.199 C56.364,121.123 51.667,114.8 48.982,107.707"
-              />
-              <path
-                id="Fill-9"
-                d="M558.509,109.788 C557.521,109.926 556.586,110.002 555.733,110.002 C547.507,110.002 542.085,105.142 542.085,93.923 L542.085,57.09 L568.068,57.09 L568.068,36.898 L542.085,36.898 L542.085,14.007 L517.404,14.007 L517.404,95.793 C517.404,121.408 531.988,133 554.051,133 C559.44,133 564.302,132.206 569.964,130.017 C562.645,124.774 559.799,116.281 558.509,109.788"
-              />
-            </g>
-            <path
-              id="Fill-11"
-              fill="#E46C42"
-              d="M695.029,116.028 C695.029,124.853 688.257,131.624 678.817,131.624 C669.377,131.624 662.605,124.853 662.605,116.028 C662.605,107.615 669.377,100.227 678.817,100.227 C688.257,100.227 695.029,107.615 695.029,116.028"
-              class="teradata-logo-period"
-            />
-          </g>
-        </svg>
-
-        <div
-          class="mat-primary mat-progress-spinner mat-spinner mat-progress-spinner-indeterminate-animation"
-          role="progressbar"
-          mode="indeterminate"
-          style="height: 100px; width: 100px;"
-        >
-          <svg
-            preserveAspectRatio="xMidYMid meet"
-            focusable="false"
-            style="height: 100px; width: 100px;"
-            viewBox="0 0 100 100"
-          >
-            <circle
-              cx="50%"
-              cy="50%"
-              style="animation-name: mat-progress-spinner-stroke-rotate-100; stroke-dasharray: 282.743px; stroke-width: 10%;"
-              r="45"
-              class="ng-star-inserted"
-            ></circle>
-          </svg>
-        </div>
-      </div>
-    </td-app>
-    <script>
-      var themeLSKey = 'vantage.theme';
-      var defaultTheme = 'light-theme';
-      localStorage.setItem(themeLSKey, localStorage.getItem(themeLSKey) || defaultTheme);
-      document.querySelector('html').classList.add(localStorage.getItem(themeLSKey));
-    </script>
+    <div id="td-pre-loader"></div>
```

Comment:
Werent we gonna try and make this dynamic? Or where did that end?

Reply:
Will be implemented in https://github.com/Teradata/vantage-ui-platform/pull/51

Reply:
Done